src/Command/ServerCommand.php

```diff
@@ -148,14 +191,23 @@ public function execute(Arguments $args, ConsoleIo $io): ?int
             $command = sprintf('%s -c %s', $command, $this->_iniPath);
         }
 
-        $command = sprintf('%s %s', $command, escapeshellarg($this->_documentRoot . '/index.php'));
-
-        $port = ':' . $this->_port;
-        $io->out(sprintf('built-in server is running in http://%s%s/', $this->_host, $port));
-        $io->out('You can exit with <info>`CTRL-C`</info>');
-        system($command);
+        return sprintf('%s %s', $command, escapeshellarg($this->_documentRoot . '/index.php'));
+    }
 
-        return static::CODE_SUCCESS;
+    /**
+     * Returns the command to run frankenphp's server.
+     *
+     * @return string
+     */
+    protected function frankenphpCommand(): string
+    {
+        return sprintf(
+            '%s php-server -a -l %s:%d -r %s',
+            (string)env('FRAKEN_PHP', 'frankenphp'),
```

Comment:
```suggestion
            (string)env('FRANKENPHP', 'frankenphp'),
```

Was the typo intentional?

Reply:
Nope, fixed